src/activity_handlers.cpp

```diff
@@ -524,25 +524,35 @@ void butchery_drops_harvest( const mtype &mt, player &p, int age, const std::fun
         const itype *drop = item::find_type( entry.drop );
 
         if( roll <= 0 ) {
-            p.add_msg_if_player( m_bad, _( "You fail to harvest: %s" ), drop->nname( 1 ).c_str() );
+            auto msg = mt.species.count( species_id( "ROBOT" ) ) ?
```

Comment:
I'd suggest a flag, something like `"MECHANICAL"`, but I'm not sure this can be done concisely in terms of PR size.